shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java

```diff
@@ -171,7 +171,7 @@ private BindingTableRule createBindingTableRule(final String bindingTableGroup)
     @Override
     public Collection<String> getAllTables() {
         Collection<String> result = new HashSet<>(getTables());
-        result.addAll(getAllActualTables());
+        result.addAll(getAllActualTables().stream().map(String::toLowerCase).collect(Collectors.toList()));
```

Comment:
@setamv Is it better to convert lowercase in the `getAllActualTables` method?

Reply:
I think so too, but getAllActualTables is referenced in other places, i don't test it. so..

Reply:
please wait and i'll test it.